cypress/components/popover-container/popover-container.cy.tsx

```diff
@@ -338,6 +358,18 @@ context("Test for Popover Container component", () => {
       }
     );
 
+    it("should call onClose callback when the escape is pressed and component is controlled", () => {
```

Comment:
```suggestion
    it("should call onClose callback when the escape is pressed", () => {
```

**suggestion** (non-blocking): I don't think it makes the test any more descriptive if it specifically refers to being controlled, as `onClose` only exists when it's controlled anyway